<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkSliceAnimationWidget</class>
 <widget class="QWidget" name="QmitkSliceAnimationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>96</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QmitkSliceAnimationWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="windowLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Window:</string>
     </property>
     <property name="buddy">
      <cstring>windowComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="windowComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>Axial</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sagittal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Coronal</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="sliceRangeLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Slice range:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="ctkRangeWidget" name="sliceRangeWidget">
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="maximum">
      <double>999.000000000000000</double>
     </property>
     <property name="maximumValue">
      <double>999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="reverseCheckBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Reverse</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkRangeWidget</class>
   <extends>QWidget</extends>
   <header>ctkRangeWidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>windowComboBox</tabstop>
  <tabstop>reverseCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
